go/nbs/table_reader.go

```diff
@@ -248,11 +258,20 @@ func (tr tableReader) readAtOffsets(readStart, readEnd uint64, reqs []getRecord,
 	}
 
 	wg.Done()
+
+	stats.BytesPerRead.Sample(readLength)
+	stats.ChunksPerRead.SampleLen(len(offsets))
```

Comment:
I forget whether `offsets` or `reqs` is a better gauge of how many chunks are being read. Does one collapse multiple reads of the same chunk into a single item?